pkg/types/container.go

```diff
@@ -93,6 +100,20 @@ type ImageStatusResponse struct {
 	Progress io.ReadCloser
 }
 
+// ImageRemoveResponse stores response when removing the image.
+type ImageRemoveResponse struct {
+	Status ImageStatus
+	Items  []ImageRemoveResponseItem
+}
+
+// ImageRemoveResponseItem stores responses' name of deleted image.
+type ImageRemoveResponseItem struct {
```

Comment:
Try using type aliasing here also.

Reply:
Please, use the name from the original package which is `ImageDeleteResponseItem`